### ISSUE TITLE
build docs for rhd

### ISSUE THREAD

**Issue by uralbash** [ITCase-django/django-unisender#5]
##

**Comment by mr-africa** [comment]
http://django-unisender.readthedocs.org/ru/latest/

Доки начаты но нужно поподробнее расписать, видимо уже не сегодня

**Comment by petrikoz** [comment]
May be use [Python Docstring](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html) and autogen from it?

**Comment by mr-africa** [comment]
Yeah docstring in future, after writing [quick start guide](http://django-unisender.readthedocs.org/ru/latest/quickstart.html)
